include/pthreadpool.h

```diff
@@ -1,6 +1,7 @@
 #ifndef PTHREADPOOL_H_
 #define PTHREADPOOL_H_
 
+#include <stdbool.h>
```

Comment:
This is unnecessary, none of the functions in the header use `bool`.

Reply:
Aaah. This is from my dev setup. Will fix.